src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -280,16 +280,16 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
           bn(10).pow(asset.precision),
         )
 
-        if (cryptoAmountBaseUnit.lt(cryptoAmountBaseUnit)) {
-          // If this throws, we know we goof'd up
-          throw new Error('Cannot send more than asset balance')
-        }
-
         const needsFeeDeduction = cryptoAmountBaseUnit
           .plus(fastFeesBaseUnit)
           .gte(assetBalanceCryptoBaseUnit)
 
-        if (needsFeeDeduction)
+        if (state?.deposit.sendMax) {
+          maybeGasDeductedCryptoAmountCryptoPrecision = bnOrZero(assetBalanceCryptoBaseUnit)
+            .minus(bn(fastFeesBaseUnit).times(TXS_BUFFER))
+            .div(bn(10).pow(asset.precision))
+            .toFixed()
```

Comment:
Ensures we're dealing with the *current* max balance and not the original one before the reconciliation Tx